reference/conanfile/methods/build.rst

```diff
@@ -0,0 +1,48 @@
+.. _reference_conanfile_methods_build:
+
+build()
+=======
+
+The ``build()`` method is used to define the build from source of the package. In practice this means calling some build system, which could be done explictly or using any of the build helpers provided by Conan:
+
+
+.. code-block:: python
+
+    from conan.tools.cmake import CMake
+
+    clas Pkg(ConanFile):
+
+        def build(self):
+            # Either using some of the Conan built-in helpers
+            cmake = CMake(self)
+            cmake.configure()  # equivalent to self.run("cmake . <other args>")
+            cmake.build() # equivalent to self.run("cmake --build . <other args>")
+            cmake.test()  # equivalent to self.run("cmake --target=RUN_TESTS")
```

Comment:
is it a right place to invoke the test? there is a dedicated `test` method

Reply:
yes, ``test()`` is exclusively for ``test_package``, not for unit tests.